light_curves/light_curve_generator.md

```diff
@@ -93,15 +93,15 @@ from gaia_functions import gaia_get_lightcurves
 from hcv_functions import hcv_get_lightcurves
 from heasarc_functions import heasarc_get_lightcurves
 from icecube_functions import icecube_get_lightcurves
-from panstarrs_functions import panstarrs_get_lightcurves
+from panstarrs_functions import panstarrs_hats_get_lightcurves
```

Comment:
Is there a reason for this change? None of the other `*_get_lightcurves` functions specify what format the data is in. Also, this change would require updating the scale_up notebook.